app/templates/email/bug-report.ts

```diff
@@ -0,0 +1,140 @@
+export const bugReportTemplates = {
+  en: `
+Describe the bug
+A clear and concise description of what the bug is. If it seems connected to some data problem (missing values, wrong parsing), please first check the cube in Cube Validator to see if everything is fine there.
+Please describe...
+
+----------------------------------------------
+To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '...'
+3. Scroll down to '...'
+4. See error
+
+----------------------------------------------
+Expected behavior
+A clear and concise description of what you expected to happen.
+Please describe...
+
+----------------------------------------------
+Screenshots or video
+If applicable, add screenshots or a short video to help put your problem into a context.
+
+----------------------------------------------
+Environment
+Please complete the following information.
+- Visualize environment and version: [e.g., INT v3.12.0]
```

Comment:
Nit: Maybe we could bump the version to most recent one (4.9.4)? I think it would look better 😅